docs/testing/component-tests.md

```diff
@@ -139,3 +139,25 @@ assert( this.themesList.props.children.length === this.props.themes.length, 'chi
 ```
 
 So here we avoid having to actually draw the `Theme` components when testing `ThemesList`.
+
+## Troubleshooting
+
+* Valid tests can fail if a component is wrapped in a higher order component, like `localize()` or `connect()`. This is because a shallow render only results in the higher component being rendered, not it's children. The best practice is to test the raw component directly, with external dependencies mocked, so that the results aren't influenced by anything outside the component being tested:
```

Comment:
`not it's children` should be `not its children` (no `'`).

Reply:
Updated.